contents/handbook/growth/sales/professional-services.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Professional services
+sidebar: Handbook
+showTitle: true
+---
+
+Some potential customers either expect to pay for professional services to help them get set up. There are others who don't ask for this, but where we can tell it would be helpful for them. 
+
+> For now this is only a service we offer to potential customers by default, so this will mainly be of interest to the [New Business](/teams/sales-cs) team.
+
+## Who we can offer professional services to
+
+A good candidate for this probably has some combination of the following:
+
+- ARR in the $100k+ region, either today or in the next 12 months (actual spend, not credit)
+- Have explicitly asked about paid implementation services and training
+- The team adopting PostHog work in person at a single location
+- They are not set up with PostHog yet, ie. we would be helping them implement PostHog in their codebase
+- They are able and willing to give our team access to their codebase
+- We are helping them get set up with session replay, feature flags, LLM analytics, and/or error tracking
```

Comment:
Startups and YC teams may fit some of this cohort and _also_ have the benefit of Hogpatch. 

How would we feel about offering this to posthog for startup users? Two ideas for how we could easily filter this down to be a less heavy lift to start with: 

- We automatically offer this to startups who have been in the startup program for 1-2 weeks but haven't used any credits yet (e.g., they're struggling to adopt us, ripe for us stepping in), or
- We just offer it to YC teams (because that's a high quality indicator and a white glove service we can offer them is great)

Reply:
Well, now I feel sad. But I also understand and appreciate the perspective.